qcodes/dataset/sqlite_base.py

```diff
@@ -271,6 +276,13 @@ def _adapt_float(fl: float) -> Union[float, str]:
     return float(fl)
 
 
+def _adapt_complex(value: Union[complex, np.complex]) -> sqlite3.Binary:
```

Comment:
Is this type annotation complete?

also, shall we make a `Complex` type as `Union[complex, np.complex, np.complex64, ...]`?

Reply:
Yes